src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/stress/StressStrategy.kt

```diff
@@ -44,19 +42,11 @@ class StressStrategy(
         }
     }
 
-    override fun run(): LincheckFailure? {
```

Comment:
Please make this refactoring (and the one in ManagedStrategy) in a separate PR

Reply:
Ok.

Reply:
Done: #262